src/centralized_telescope/params.rs

```diff
@@ -1,16 +1,147 @@
-//! ALBA's Setup structure
+//! Centralized Telescope's Params structure comprising the internal parameters
```

Comment:
I wonder if we should add some sort of range checks for some of the parameters - not super urgent, but a good practice.

Reply:
We will add unit tests as done for the lottery params! See issue #128